MedscribeUI/src/components/Utilities/ThreeBtnSelector.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from '@mantine/core';
+import { useState, useEffect } from 'react';
+
+interface ThreeBtnSelectorProps {
+  buttonLabelOptions: string[];
+  initialSelected?: string;
+  onSelect: (value: string) => void;
```

Comment:
Yea my bad take this out. We wont be needing this. We only need this component to update the current pronoun that it is selected.

Reply:
ignore this